python-sdk/src/astro/sql/operators/raw_sql.py

```diff
@@ -93,16 +93,21 @@ def make_row_serializable(rows: Any) -> Any:
         """
         if not settings.NEED_CUSTOM_SERIALIZATION:
             return rows
+        if isinstance(rows, PandasDataframe):
+            return rows
         if isinstance(rows, Iterable):
             return [SdkLegacyRow.from_legacy_row(r) if isinstance(r, SQLAlcRow) else r for r in rows]
         return rows
 
     @staticmethod
-    def results_as_list(result: ResultProxy) -> list:
+    def results_as_list(results: ResultProxy) -> list:
         """
         Convert the result of a SQL query to a list
         """
-        return cast(List[Any], result.fetchall())
+        data = []
+        for result in results.fetchall():
+            data.append(result.values())
```

Comment:
SHould this be:

```suggestion
        for result, _ in results.fetchall():
            data.append(result.values())
```

Reply:
I think this may change from database to database. I'll extent the testcase to cover all DB

Reply:
@kaxil I think it should be `for result in results.fetchall():` only, wrote test case for all DBs and its working for all.

Reply:
`result.values()` expects that the value returned will be a dict always? or am I mis-understanding

Reply:
discussed on slack.